namui/src/namui/common/types/time.rs

```diff
@@ -5,18 +5,53 @@ use ordered_float::OrderedFloat;
 pub enum Time {
     Ms(f32),
     Sec(f32),
-    Min(f32),
+    Minute(f32),
     Hour(f32),
     Day(f32),
     Week(f32),
 }
 
+pub trait TimeExt {
+    fn ms(self) -> Time;
+    fn sec(self) -> Time;
+    fn minute(self) -> Time;
```

Comment:
The reason why I change `min` to `minute` is becuase `1.min()` conflicts with math's min(imum) and min(ute).